runtime/Makefile

```diff
@@ -150,14 +150,18 @@ critest:
 		--volume /dev:/dev \
 		--volume /run/udev/control:/run/udev/control \
 		--volume $(CURDIR)/..:/src \
+		--volume $(CURDIR)/../examples/etc/containerd/firecracker-runtime.json:/etc/containerd/firecracker-runtime.json \
+		--volume $(CURDIR)/../tools/demo/fc-br0.interface:/etc/network/interfaces.d/fc-br0 \
+		--volume $(CURDIR)/logs:/var/log/firecracker-containerd-test \
 		--volume $(GO_CACHE_VOLUME_NAME):/go \
 		--env FICD_DM_VOLUME_GROUP=$(FICD_DM_VOLUME_GROUP) \
 		--env FICD_DM_POOL=$(FICD_DM_POOL) \
 		--env GOPROXY=direct \
 		--env GOSUMDB=off \
+      	--env ACK_GINKGO_DEPRECATIONS=1.16.5 \
 		--workdir="/src/runtime" \
 		$(FIRECRACKER_CONTAINERD_TEST_IMAGE):$(DOCKER_IMAGE_TAG) \
-		"critest -runtime-endpoint unix:///run/firecracker-containerd/containerd.sock"
+		"sleep 1 && critest -runtime-endpoint unix:///run/firecracker-containerd/containerd.sock"
```

Comment:
Why does it need to sleep?

Reply:
This is to allow fc-cd to be fully up and running before running CRI tests. Before, the first test or two would fail with connection issues due to fc-cd not having started listening for incoming requests.